hotelReservation/helm-chart/hotelreservation/charts/wrk2/values.yaml

```diff
@@ -0,0 +1,11 @@
+wrk2:
+  replicas: 1
```

Comment:
This is referenced in `wrk2_deployment.tpl` but I don't know where it was being set before.